docs/alternativesemantics.md

```diff
@@ -0,0 +1,202 @@
+# Alternative semantics for Verona.
+
+## Abstract
+
+This document is a thought experiment in how runtime support can reduce the complexity of the type system.  The key insight is to delay certain operations to a point where we can trivially know that there is no aliasing.  Aliases into a region are difficult to track accurately at compile time, by relaxing the semantics of when a programmer can assume an event occurs allows for a simpler type system.

```

Comment:
> ...compile time, by relaxing... 
This sentence is a bit garbled . Maybe should be a new sentence here without 'by'?